### DISCUSSION TITLE
react-native-reanimated error while installing

### DISCUSSION THREAD

**Discussion by KedarKK1** [react-navigation/react-navigation#10837] [Q&A]
Getting an error when installing  react-native-reanimated

> Configure project :react-native-reanimated
Native libs debug enabled: false
Android gradle plugin: 7.1.1
Gradle: 7.3.3
building Reanimated2
WARNING:Software Components will not be created automatically for Maven publishing from Android Gradle Plugin 8.0. To opt-in to the future behavior, set the Gradle property android.disableAutomaticComponentCreation=true in the `gradle.properties` file or use the new publishing DSL.
[CXX1101] NDK at C:\Users\ABC\AppData\Local\Android\Sdk\ndk\21.4.7075529 did not have a source.properties file
Observed package id 'extras;intel;Hardware_Accelerated_Execution_Manager' in inconsistent location 'C:\Users\ABC\AppData\Local\Android\Sdk\extras\intel\Hardware_Accelerated_Execution_Manager-2' (Expected 'C:\Users\ABC\AppData\Local\Android\Sdk\extras\intel\Hardware_Accelerated_Execution_Manager')
Already observed package id 'extras;intel;Hardware_Accelerated_Execution_Manager' in 'C:\Users\ABC\AppData\Local\Android\Sdk\extras\intel\Hardware_Accelerated_Execution_Manager'. Skipping duplicate at 'C:\Users\ABC\AppData\Local\Android\Sdk\extras\intel\Hardware_Accelerated_Execution_Manager-2'
Warning: Observed package id 'extras;intel;Hardware_Accelerated_Execution_Manager' in inconsistent location 'C:\Users\ABC\AppData\Local\Android\Sdk\extras\intel\Hardware_Accelerated_Execution_Manager-2' (Expected 'C:\Users\ABC\AppData\Local\Android\Sdk\extras\intel\Hardware_Accelerated_Execution_Manager')  
Warning: Already observed package id 'extras;intel;Hardware_Accelerated_Execution_Manager' in 'C:\Users\ at 'C:\Users\ABC\AppData\Local\Android\Sdk\extras\intel\Hardware_Accelerated_Execution_Manager-2'    
Warning: Observed package id 'extras;intel;Hardware_Accelerated_Execution_Manager' in inconsistent location 'C:\Users\ABC\AppData\Local\Android\Sdk\extras\intel\Hardware_Accelerated_Execution_Manager-2' (Expected 'C:\Users\ABC\AppData\Local\Android\Sdk\extras\intel\Hardware_Accelerated_Execution_Manager')  
Warning: Already observed package id 'extras;intel;Hardware_Accelerated_Execution_Manager' in 'C:\Users\ABC\AppData\Local\Android\Sdk\extras\intel\Hardware_Accelerated_Execution_Manager'. Skipping duplicate at 'C:\Users\ABC\AppData\Local\Android\Sdk\extras\intel\Hardware_Accelerated_Execution_Manager-2'    
Checking the license for package NDK (Side by side) 21.4.7075529 in C:\Users\ABC\AppData\Local\Android\Sdk\licenses
License for package NDK (Side by side) 21.4.7075529 accepted.
Preparing "Install NDK (Side by side) 21.4.7075529 (revision: 21.4.7075529)".
<========-----> 62% CONFIGURING [3m 6s]
> :react-native-reanimated
Terminate batch job (Y/N)? Y
PS C:\Users\ABC\Documents\ABC\PICT\Internship\Trying React-navigations\mynavs\android> cd ..        
PS C:\Users\ABC\Documents\ABC\PICT\Internship\Trying React-navigations\mynavs> npx react-native run-android
info Running jetifier to migrate libraries to AndroidX. You can disable it using "--no-jetifier" flag.
Jetifier found 1236 file(s) to forward-jetify. Using 8 workers...
info Starting JS server...
info Launching emulator...
error Failed to launch emulator. Reason: Could not start emulator within 30 seconds..
warn Please launch an emulator manually or connect a device. Otherwise app may fail to launch.
info Installing the app...
Starting a Gradle Daemon (subsequent builds will be faster)

> Configure project :react-native-reanimated
Native libs debug enabled: false
Android gradle plugin: 7.1.1
Gradle: 7.3.3
building Reanimated2
WARNING:Software Components will not be created automatically for Maven publishing from Android Gradle Plugin 8.0. To opt-in to the future behavior, set the Gradle property android.disableAutomaticComponentCreation=true in the `gradle.properties` file or use the new publishing DSL.
Observed package id 'extras;intel;Hardware_Accelerated_Execution_Manager' in inconsistent location 'C:\Users\ABC\AppData\Local\Android\Sdk\extras\intel\Hardware_Accelerated_Execution_Manager-2' (Expected 'C:\Users\ABC\AppData\Local\Android\Sdk\extras\intel\Hardware_Accelerated_Execution_Manager')
Already observed package id 'extras;intel;Hardware_Accelerated_Execution_Manager' in 'C:\Users\ABC\AppData\Local\Android\Sdk\extras\intel\Hardware_Accelerated_Execution_Manager'. Skipping duplicate at 'C:\Users\ABC\AppData\Local\Android\Sdk\extras\intel\Hardware_Accelerated_Execution_Manager-2'
Warning: Observed package id 'extras;intel;Hardware_Accelerated_Execution_Manager' in inconsistent location 'C:\Users\ABC\AppData\Local\Android\Sdk\extras\intel\Hardware_Accelerated_Execution_Manager-2' (Expected 'C:\Users\ABC\AppData\Local\Android\Sdk\extras\intel\Hardware_Accelerated_Execution_Manager')  
Warning: Already observed package id 'extras;intel;Hardware_Accelerated_Execution_Manager' in 'C:\Users\ABC\AppData\Local\Android\Sdk\extras\intel\Hardware_Accelerated_Execution_Manager'. Skipping duplicate at 'C:\Users\ABC\AppData\Local\Android\Sdk\extras\intel\Hardware_Accelerated_Execution_Manager-2'    
Warning: Observed package id 'extras;intel;Hardware_Accelerated_Execution_Manager' in inconsistent location 'C:\Users\ABC\AppData\Local\Android\Sdk\extras\intel\Hardware_Accelerated_Execution_Manager-2' (Expected 'C:\Users\ABC\AppData\Local\Android\Sdk\extras\intel\Hardware_Accelerated_Execution_Manager')  
Warning: Already observed package id 'extras;intel;Hardware_Accelerated_Execution_Manager' in 'C:\Users\ABC\AppData\Local\Android\Sdk\extras\intel\Hardware_Accelerated_Execution_Manager'. Skipping duplicate at 'C:\Users\ABC\AppData\Local\Android\Sdk\extras\intel\Hardware_Accelerated_Execution_Manager-2'    
Checking the license for package NDK (Side by side) 21.4.7075529 in C:\Users\ABC\AppData\Local\Android\Sdk\licenses
License for package NDK (Side by side) 21.4.7075529 accepted.
Preparing "Install NDK (Side by side) 21.4.7075529 (revision: 21.4.7075529)".
<========-----> 62% CONFIGURING [3m 31s]
> :react-native-reanimated

**Comment by KedarKK1** [comment]
I waited for some time (>16+ minutes), then it worked perfectly !!!!
This is not an error, just takes some time to took off

**Comment by r-hirsch** [comment]
As you said, it hadn't finished yet. Just for future reference if you look at the second to last line you posted you can see a progess bar (62%) and the time that has elapsed so far (3 minutes, 31 seconds).
